sycl/source/detail/device_impl.cpp

```diff
@@ -608,6 +608,9 @@ bool device_impl::has(aspect Aspect) const {
             sizeof(ur_bool_t), &support, nullptr) == UR_RESULT_SUCCESS;
     return call_successful && support;
   }
+  case aspect::ext_oneapi_bindless_images_gather: {
+    return this->getBackend() == backend::ext_oneapi_cuda;
```

Comment:
Is there a UR query we could use instead? We should try to avoid checking backends as much as possible.

Reply:
Yes totally makes sense to add this now. Done.